Example/Custom Integration/FPXExampleViewController.m

```diff
@@ -123,6 +126,27 @@ - (void)pay {
                                                }];
     }];
 }
-#pragma clang diagnostic pop
+
+- (void)bankSelectionViewController:(nonnull STPBankSelectionViewController *)bankViewController didCreatePaymentMethodParams:(STPPaymentMethodParams *)paymentMethodParams {
+    [self payWithBankAccount:paymentMethodParams];
+}
+
+#pragma mark - STPAuthenticationContext
+
+- (UIViewController *)authenticationPresentingViewController {
+    return self.navigationController.topViewController;
+}
+
+- (void)authenticationContextWillDismissViewController:(UIViewController *)viewController {
+    // If we're launching directly into the SFSafariViewController for authentication from the bank selector,
+    // we'll want to remove the bank selector from the VC stack and pop back directly to our main controller on dismiss.
```

Comment:
supernit: as a user I might not understand why we want to do this (is it necessary? what happens if I don't do it?)
Suggestion: "// remove the bank selector from the VC stack and pop back directly to our main controller on dismiss for better UX".